test/views/TaxYearSummaryViewSpec.scala

```diff
@@ -181,15 +181,15 @@ class TaxYearSummaryViewSpec extends ViewSpec with FeatureSwitching {
     val estimate: String = s"6 April ${testYear - 1} to 1 January 2020 estimate"
     val totalDue: String = "Total Due"
     val taxDue: String = "£4.04"
-    val calcDateInfo: String = s"This is only based on figures we’ve already received to $calcDate. It’s not your final tax bill."
+    val calcDateInfo: String = "This is not your final tax bill, it's only based on figures we've already received."
     val calcEstimateInfo: String = "This is a year to date estimate based on figures we’ve already received."
     val taxCalculation: String = s"6 April ${testYear - 1} to 5 January $testYear calculation"
     val taxCalculationHeading: String = "Calculation"
     val taxCalculationTab: String = "Calculation"
     val taxCalculationNoData: String = "No calculation yet"
     val forecastNoData: String = "No forecast yet"
     val forecastNoDataNote: String = "You will be able to see your forecast for the whole year once you have sent an update."
-    val unattendedCalcPara: String = "! We have updated the calculation for you. You can see more details in your record-keeping software."
+    val unattendedCalcPara: String = "! We've updated the calculation for you. Check your record-keeping software for more details."
```

Comment:
same

Reply:
Thanks, But we have a Tech Debt ticket to fix all of those messages in one go.